index.js

```diff
@@ -15,6 +15,16 @@ const SPORT_TYPES = {
   run: '1',
   walk: '19',
   weight_training: '34',
+  swim: '18',
+  //there is no pilates,tennis, mountain_bike
+  pilates: '31',
+  //no tennis either (23)
+  tennis: '31',
+  mountain_bike: '4',
+  other: '5',
+  hike: '7',
+  cycling_ergometer: '15',
```

Comment:
Can you check if the numbers are right? This seem to be a duplicate with `indoor_ride`.

Reply:
I mentioned in PR comment:

> there is a typo, please fix it, should be:
> 
> tennis: '23',